drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/index/CellBuilder.java

```diff
@@ -61,14 +61,23 @@ public void build( final Rule rule,
 
         } else if ( baseColumn instanceof ConditionCol52 ) {
 
-            new FieldConditionsBuilder( index,
-                                        model,
-                                        rule,
-                                        row,
-                                        utils,
-                                        getPattern( rule ) ).buildCondition( ( ConditionCol52 ) baseColumn,
-                                                                             columnIndex );
 
+            if ( baseColumn instanceof BRLConditionVariableColumn ) {
+                new BRLConditionsBuilder( index,
+                                          model,
+                                          rule,
+                                          row,
+                                          utils ).buildCondition( ( BRLConditionVariableColumn ) baseColumn,
+                                                                  columnIndex );
+            } else {
+                new FieldConditionsBuilder( index,
+                                            model,
+                                            rule,
+                                            row,
+                                            utils,
+                                            getPattern( rule ) ).buildCondition( ( ConditionCol52 ) baseColumn,
+                                                                                 columnIndex );
+            }
```

Comment:
This caused the NPE. There is not pattern to be found for the new BRL column, still I tried to build the `FieldCondition` that for fields of a pattern.